academy/ibc/migration.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "IBC Migration"
+order: 
+description: 
+tag: deep-dive
+---
+
+# IBC Migration
+
+When working with IBC, it is important to be able to upgrade IBC chains and clients. IBC-connected chains must perform an IBC upgrade if their upgrade will break counterparty IBC clients. This is of importance for all developers working with Tendermint clients enabling upgrades.
```

Comment:
what is the difference between a migration and upgrade if any?  why use migration for the title if you are talking about upgrades after?

Reply:
I have a conceptual problem with sentence 2. Is the intention this:

"a connected chain **which is not IBC and which is upgraded in such a way as to conflict with IBC** must also perform an IBC upgrade to prevent breaking counterparty IBC clients"?

I can do better than that sentence (!) but I want to make sure I understand the intended meaning before I edit.